src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -12394,6 +12410,7 @@ def resample(
                     numeric_only=resample_method_kwargs.get("numeric_only", False),
                     is_series_groupby=is_series,
                 )
+
```

Comment:
```suggestion
```

Reply:
This was intentional – there is a big block of case statements right before this so having the extra line helps with readability :)